src/app/api/v1/admin/users/[fxaUid]/route.ts

```diff
@@ -63,7 +61,11 @@ export async function GET(
 
       const fxaUid = params.fxaUid;
       const subscriber = await getSubscriberByFxaUid(fxaUid);
-      if (!subscriber) {
+      if (
+        !subscriber ||
+        (process.env.APP_ENV === "production" &&
+          !isMozMail(subscriber.primary_email))
```

Comment:
I think I'd separate this out into a separate `if` clause (that returns a 403 Forbidden, if you want to be pure about it), just to reduce the complexity of the disjunction, to make sure we're absolutely explicit about who's allowed to call this when, and to avoid us making mistakes about that in the future.

(Oh, and maybe instead of comparing to `"production"`, check if it's not `"stage"` or `"local"`, i.e. such that it fails if it's not set or set to an invalid value.)

Reply:
Good idea. I’m explicitly returning a `403` in https://github.com/mozilla/blurts-server/commit/0ac6234b96f03678a0751e6bfe9c6e3656deeb63.